docs/src/main/asciidoc/migrations.adoc

```diff
@@ -111,32 +116,42 @@ task copyStubs(type: Copy, dependsOn: 'generateWireMockClientStubs') {
 }
 ----
 
+[[cloud-verifier-1.1-1.2]]
 === 1.1.x -> 1.2.x
 
+This section covers upgrading from version 1.1 to version 1.2.
+
 ==== Custom `HttpServerStub`
 
-By introducing a method `String registeredMappings()` in the public interface
-`HttpServerStub`, if you wrote a custom `HttpServerStub` implementation, you'll
-have to implement that method too. It should return a `String` representing
-all mappings available in a single `HttpServerStub`. Related to
-https://github.com/spring-cloud/spring-cloud-contract/issues/355[issue 355].
+`HttpServerStub` includes a method that was not in version 1.1. The method is
+`String registeredMappings()` If you have classes that implement `HttpServerStub`, you
+now have to implement the `registeredMappings()` method. It should return a `String`
+representing all mappings available in a single `HttpServerStub`.
+
+See https://github.com/spring-cloud/spring-cloud-contract/issues/355[issue 355] for more
+detail.
 
 ==== New packages for generated tests
 
 The flow for setting the generated tests package name will look like this:
 
-- pick `basePackageForTests`
-- if `basePackageForTests` wasn't set pick the package from `baseClassForTests`
-- if `baseClassForTests` wasn't set pick `packageWithBaseClasses`
-- if nothing got set pick the default `org.springframework.cloud.contract.verifier.tests` value
+* Set `basePackageForTests`
+* If `basePackageForTests` was not set, pick the package from `baseClassForTests`
+* If `baseClassForTests` was not set, pick `packageWithBaseClasses`
+* If nothing got set, pick the default value:
+`org.springframework.cloud.contract.verifier.tests`
 
-Related to
-https://github.com/spring-cloud/spring-cloud-contract/issues/260[issue 260].
+See https://github.com/spring-cloud/spring-cloud-contract/issues/260[issue 260] for more
+detail.
 
 ==== New methods in TemplateProcessor
 
-In order to add support for `fromRequest.path` some methods had to be added to the
+In order to add support for `fromRequest.path`, some methods had to be added to the
 `TemplateProcessor` interface.
+// TODO What methods?
```

Comment:
Ooops... :P

the missing methods are `path()` and `path(int index)`

Reply:
Thank you. I'll add that information.